web/pipeline/models.py

```diff
@@ -1,27 +1,53 @@
-from django.db import models
+from django.contrib.gis.db import models
 from django.contrib.gis.db.models import PointField
 
 
 class Location(models.Model):
     name = models.CharField(null=True, blank=True, max_length=255)
     point = PointField(null=True, blank=True)
+    # location_type = models.CharField(null=True, blank=True, max_length=255)
+    # source = models.CharField(null=True, blank=True, max_length=255)
+    # kind = models.CharField(null=True, blank=True, max_length=255)
 
     def __str__(self):
         return self.name
 
     def latitude(self):
-        return self.location[1]
+        return self.point[1]
 
     def longitude(self):
-        return self.location[0]
+        return self.point[0]
+
+
+class Hospital(Location):
```

Comment:
A Hospital is a child model of Location, so it also includes the `name` and `point` fields from Location